Transformations:
  - frame_chain: lidar_right_frame, lidar_left_frame
    invert_chain: [1]
    path_chain:  /media/timrehbronn/DAS1/G-Nav/Deneb/Paper/eval_logs/calib_log_2024-09-30_13-31-33_rslidar_points_r_rslidar_points_l.json

Reference Frame:

remappings:
  - input_topic: /rslidar_points_l
    output_topic: /lidar_left
    new_frame_id: lidar_left_frame
  - input_topic: /rslidar_points_r
    output_topic: /lidar_right
    new_frame_id: lidar_right_frame

